SdarotAPI/SdarotDriver.cs

```diff
@@ -14,9 +18,18 @@ public static async Task<string> RetrieveSdarotDomain()
 
     public SdarotDriver(bool ignoreChecks)
     {
+        HttpClientHandler handler = new HttpClientHandler
+        {
+            AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
+        };
+
+        _httpClient = new HttpClient(handler);
```

Comment:
If you create the `HttpClient` here, you can remove it's initialization from line 9

Reply:
You mean the current initialization at the declaration (line 5)?

Reply:
Yes, since the using statements has moved